Gruntfile.js

```diff
@@ -53,7 +88,8 @@ module.exports = function (grunt) {
 
 	grunt.registerTask('dist', [
 		'typings',
-		'tslint',
+		// TODO: uncomment this, figure out a way to ignore generated CSS TS modules
```

Comment:
Added `/* tslint:disable:object-literal-key-quotes quotemark whitespace */` to the beginning of each generated file so we could enable the `tslint` task again.